samples/OpenIdConnectSample/Startup.cs

```diff
@@ -28,8 +29,10 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
             app.UseOpenIdConnectAuthentication(options =>
             {
                 options.ClientId = "63a87a83-64b9-4ac1-b2c5-092126f8474f";
+                options.ClientSecret = "Yse2iP7tO1Azq0iDajNisMaTSnIDv+FXmAsFuXr+Cy8=";
```

Comment:
Understood :smile: 

Is it worth adding a comment indicating that `client_secret` is not used when specifying `response_type=id_token`/`response_type=id_token token`?